modules/m4d-template/values.yaml

```diff
@@ -5,4 +5,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+metadata:
+  name: "hello-world"
+  namespace: "default"
```

Comment:
Namespace should not be defined - it is taken from the blueprint during orchestration.